agent/src/main/java/com/code_intelligence/jazzer/runtime/TraceDataFlowNativeCallbacks.java

```diff
@@ -57,7 +57,7 @@
 
   /* indirect-calls */
   // Calls: void __sanitizer_cov_trace_pc_indir(uintptr_t Callee);
-  private static native void tracePcIndir(int callee, int caller);
+  public static native void tracePcIndir(int callee, int caller);
```

Comment:
Could we switch the order of the parameters to keep them in sync with the libFuzzer's implementation `void TracePC::HandleCallerCallee(uintptr_t Caller, uintptr_t Callee)`?

Reply:
I thought about that at first, but it would mean breaking consistency with all the other Java PC hooks, which pass `id` in last. Should I perhaps also move those `id` arguments to the beginning of the argument list?

Reply:
I don't think it is that urgent but caused some confusion for me during the review. Moving the `id` arguments at the beginning of the argument list is a good idea and guarantees that the remaining arguments can be kept in sync with the corresponding ones for the libFuzzer hooks. As I said, it is nice to have but not very urgent.